packages/css-engine/src/core/atomic.test.ts

```diff
@@ -175,3 +175,33 @@ test("distinct similar declarations from different breakpoints", () => {
 }"
 `);
 });
+
+test("support descendent suffix", () => {
+  const sheet = createRegularStyleSheet();
+  sheet.addMediaRule("x");
+  const rule1 = sheet.addNestingRule("instance");
+  rule1.setDeclaration({
+    breakpoint: "x",
+    selector: ":hover",
+    property: "display",
+    value: { type: "keyword", value: "block" },
+  });
+  const rule2 = sheet.addNestingRule("instance", " img");
```

Comment:
are you sure you want to have space and such as part of the selector string and not as something completely separated?

Reply:
Yep, string is enough